haproxy/README.md

```diff
@@ -12,9 +12,17 @@ Capture HAProxy activity in Datadog to:
 
 ## Setup
 
+### Prerequisites
+
+This OpenMetrics-based integration has a latest version (V2) and a legacy version (V1). To get all the most up-to-date features, Datadog recommends upgrading to the latest version (enterprise version 1.9rc1 or later) to enable the Prometheus endpoint on HAProxy. If you are using the legacy version, use the [HAProxy Prometheus exporter][3], or set up the [socket-based integration](#using-the-stats-endpoint). For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][29].
+
+The `use_openmetrics` option uses [OpenMetrics V2][26] for metric collection, which requires Agent v7.35 or later, or for you to [enable Python 3][27] in Agent v6.35 or later. For hosts that are unable to use Python 3 or are on Agent v7.34 or earlier, use the OpenMetrics V1 implementation or the [socket-based legacy integration](#using-the-stats-endpoint). 
+
+Metrics marked as `[OpenMetrics V1]` or `[OpenMetrics V2]` are only available using the corresponding version of the HAProxy integration. Metrics marked as `[OpenMetrics V1 and V2]` are collected by both versions.
```

Comment:
```suggestion
To use Prometheus/OpenMetrics v1 instead of OpenMetrics v2, change the `use_openmetrics` option to `use_prometheus`, and change the `openmetrics_endpoint` option to `prometheus_url`. For more information, see the [Prometheus and OpenMetrics metrics collection from a host documentation][29]. Metrics marked as `[OpenMetrics V1]` or `[OpenMetrics V2]` are only available using the corresponding version of the HAProxy integration. Metrics marked as `[OpenMetrics V1 and V2]` are collected by both versions.
```

Reply:
I would leave this as it was (just the reference to "tags" in metric descriptions), and move the additional information to the configuration section (somewhere in [here](https://github.com/DataDog/integrations-core/blob/0fbfb5a381b96539c2143da8464d25a5985a812d/haproxy/README.md?plain=1#L29)).